Content.Server/Body/Components/MetabolizerComponent.cs

```diff
@@ -55,12 +55,12 @@ public sealed partial class MetabolizerComponent : Component
         public bool RemoveEmpty = false;
 
         /// <summary>
-        ///     How many reagents can this metabolizer process at once?
+        ///     How many Poison can this metabolizer process at once?
         ///     Used to nerf 'stacked poisons' where having 5+ different poisons in a syringe, even at low
         ///     quantity, would be muuuuch better than just one poison acting.
         /// </summary>
         [DataField("maxReagents")]
-        public int MaxReagentsProcessable = 3;
+        public int MaxPoisonProcessable = 3;
```

Comment:
Nitpick - this should be plural.
```suggestion
        public int MaxPoisonsProcessable = 3;
```